llvm/docs/LangRef.rst

```diff
@@ -16723,21 +16723,29 @@ type.
 
 Semantics:
 """"""""""
+Follows the semantics of minimumNumber in IEEE-754-2019, except for signaling NaNs. If either operand
+is an sNaN, the result is always a qNaN. This matches the recommended behavior for the libm
```

Comment:
I can't decide if this is the correct way to describe it, relative to IEEE. It's IEEE-754-2008 minNum with stronger -0 behavior. This is probably clearer, given this is where the intrinsic got its name from

Reply:
I also prefer describe it with IEEE-754-2008 as base.